swift/README.md

```diff
@@ -37,7 +37,7 @@ class Component {
 How do I handle errors in Swift?
 ================================
 
-Swift doesn't have exceptions, so errors must somehow be returned to your caller.
+As of Swift 2.0 exceptions have been introduced into the language and many of the built-in APIs make use of them. We have decided to stick with our existing pattern of returning errors.
```

Comment:
Can we clarify that the APIs that use this are all the ones that used to use `someMethod(..., error: *NSError)` ?  People coming from Cocoa will be able to identify with that easily, I think.